contracts/test/RFQ.t.sol

```diff
@@ -293,65 +293,65 @@ contract RFQTest is StrategySharedSetup {
 
     function testFillDAIToUSDT_EOAUserAndEOAMaker() public {
         RFQLibEIP712.Order memory order = DEFAULT_ORDER;
-        bytes memory makerSig = _signOrder(makerPrivateKey, order, SignatureValidator.SignatureType.EIP712);
-        bytes memory userSig = _signFill(userPrivateKey, order, SignatureValidator.SignatureType.EIP712);
-        bytes memory payload = _genFillPayload(order, makerSig, userSig);
+        bytes memory makerSig = _signOrder({ privateKey: makerPrivateKey, order: order, sigType: SignatureValidator.SignatureType.EIP712 });
+        bytes memory userSig = _signFill({ privateKey: userPrivateKey, order: order, sigType: SignatureValidator.SignatureType.EIP712 });
+        bytes memory payload = _genFillPayload({ order: order, makerSig: makerSig, userSig: userSig });
 
-        BalanceSnapshot.Snapshot memory userTakerAsset = BalanceSnapshot.take(user, order.takerAssetAddr);
-        BalanceSnapshot.Snapshot memory receiverMakerAsset = BalanceSnapshot.take(receiver, order.makerAssetAddr);
-        BalanceSnapshot.Snapshot memory makerTakerAsset = BalanceSnapshot.take(maker, order.takerAssetAddr);
-        BalanceSnapshot.Snapshot memory makerMakerAsset = BalanceSnapshot.take(maker, order.makerAssetAddr);
+        BalanceSnapshot.Snapshot memory userTakerAsset = BalanceSnapshot.take({ owner: user, token: order.takerAssetAddr });
+        BalanceSnapshot.Snapshot memory receiverMakerAsset = BalanceSnapshot.take({ owner: receiver, token: order.makerAssetAddr });
+        BalanceSnapshot.Snapshot memory makerTakerAsset = BalanceSnapshot.take({ owner: maker, token: order.takerAssetAddr });
+        BalanceSnapshot.Snapshot memory makerMakerAsset = BalanceSnapshot.take({ owner: maker, token: order.makerAssetAddr });
 
         vm.prank(user, user); // Only EOA
         userProxy.toRFQ(payload);
 
-        userTakerAsset.assertChange(-int256(order.takerAssetAmount));
-        receiverMakerAsset.assertChange(int256(order.makerAssetAmount));
-        makerTakerAsset.assertChange(int256(order.takerAssetAmount));
-        makerMakerAsset.assertChange(-int256(order.makerAssetAmount));
+        userTakerAsset.assertChange({ expectedChange: -int256(order.takerAssetAmount) });
+        receiverMakerAsset.assertChange({ expectedChange: int256(order.makerAssetAmount) });
+        makerTakerAsset.assertChange({ expectedChange: int256(order.takerAssetAmount) });
+        makerMakerAsset.assertChange({ expectedChange: -int256(order.makerAssetAmount) });
```

Comment:
nit: I think the function name is already presenting enough meaning for reader, maybe only one argument case could get rid of named parameter?

Reply:
Sure, I think it can be got rid when understand the usage of Balance Snapshot.Snapshot.
What do others think?

Reply:
Yeah, we only need it for functions with multiple parameters.